drape/font_texture.cpp

```diff
@@ -184,6 +206,19 @@ ref_ptr<Texture::ResourceInfo> GlyphIndex::MapResource(GlyphKey const & key, boo
   return make_ref(&res.first->second);
 }
 
+bool GlyphIndex::CanBeGlyphPacked(uint32_t glyphsCount) const
+{
+  if (glyphsCount == 0)
+    return true;
+
+  if (m_packer.IsFull())
+    return false;
+
+  float const kGlyphScalar = 1.5f;
+  uint32_t const baseSize = static_cast<uint32_t>(m_mng->GetBaseGlyphHeight() * kGlyphScalar);
```

Comment:
Why do you need floating point calculations here? m_mng->GetBaseGlyphHeight() * 3 / 2?